File: src/main/scala/com/chinawiserv/wsmp/WsmpBooter.scala
package com.chinawiserv.wsmp

import com.chinawiserv.wsmp.kafka.KafkaReader

object WsmpBooter {

  def main(args: Array[String]): Unit = {
    val kafkaReader = new KafkaReader("172.16.7.202:9092,172.16.7.203:9092,172.16.7.204:9092", "dom");
    kafkaReader.start();
    kafkaReader.waitToStop();
  }

}

File: src/main/java/com/chinawiserv/wsmp/WsmpBooter.java
package com.chinawiserv.wsmp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration;
import org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication(
	exclude = {
			JmxAutoConfiguration.class,
			MongoAutoConfiguration.class,
	}	
)
@EnableAsync
public class WsmpBooter {

	public static void main(String[] args) {
		SpringApplication.run(WsmpBooter.class, args);
	}
}
